src/test/java/com/rocketden/main/mapper/GameMapperTests.java

```diff
@@ -58,13 +65,60 @@ public void toDto() {
         Room room = new Room();
         room.setRoomId(ROOM_ID);
 
-        Game game = new Game();
-        game.setRoom(room);
+        User user = new User();
+        user.setNickname(NICKNAME);
+        user.setUserId(USER_ID);
+        room.addUser(user);
+
+        Game game = GameMapper.fromRoom(room);
+
+        PlayerCode playerCode = new PlayerCode();
+        playerCode.setCode(CODE);
+        playerCode.setLanguage(LANGUAGE);
+
+        Submission submission = new Submission();
+        submission.setPlayerCode(playerCode);
+        submission.setNumTestCases(TEST_CASES);
+        submission.setNumCorrect(TEST_CASES);
+
+        Player player = game.getPlayers().get(USER_ID);
+        player.setSolved(true);
+        player.setPlayerCode(playerCode);
+        player.getSubmissions().add(submission);
 
         GameDto gameDto = GameMapper.toDto(game);
 
-        assertEquals(RoomMapper.toDto(room), gameDto.getRoomDto());
-        // Assert player map is null for now until implemented
-        assertNull(gameDto.getPlayerMap());
+        assertEquals(RoomMapper.toDto(room), gameDto.getRoom());
+        assertEquals(1, gameDto.getPlayers().size());
+
+        PlayerDto playerDto = gameDto.getPlayers().get(0);
+        assertEquals(UserMapper.toDto(user), playerDto.getUser());
+        assertEquals(player.getSolved(), playerDto.getSolved());
+        assertEquals(playerCode.getCode(), playerDto.getCode());
+        assertEquals(playerCode.getLanguage(), playerDto.getLanguage());
+        assertEquals(1, playerDto.getSubmissions().size());
+
+        SubmissionDto submissionDto = playerDto.getSubmissions().get(0);
+        assertEquals(submission.getPlayerCode().getCode(), submissionDto.getCode());
+        assertEquals(submission.getPlayerCode().getLanguage(), submissionDto.getLanguage());
```

Comment:
If you end up changing `submissionDto` to hold a `playerCode` attribute, you could simply have one `assertEquals` statement?

Reply:
Gave my reasoning behind not using `PlayerCode` for this - lmk what you think and I can change this too if we decide on using it